static/css/styles.css

```diff
@@ -8,33 +8,33 @@
     --united-violet: 139, 0, 255;
 }
 
+/* General Reset */
 * {
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-    font-family: Arial, sans-serif;
     text-decoration: none !important;
 }
 
+/* Base Font Family */
 body,
 html {
+    font-family: 'Lato', sans-serif;
     height: 100%;
 }
 
 body:not(#bg-white) {
-    /* https://images.pexels.com/photos/1167034/pexels-photo-1167034.jpeg */

```

Comment:
If you're removing the comments for credits, then it must be added to the README for attributions, so be sure to pass this on to @hannahro15

Reply:
This wasn't intentional - I'm uncertain what I need to add/remove/revert to resolve this... please further advise?